src/mds/MDSDaemon.cc

```diff
@@ -664,6 +664,13 @@ const std::vector<MDSDaemon::MDSCommand>& MDSDaemon::get_commands()
         "name=heapcmd,type=CephChoices,strings=dump|start_profiler|stop_profiler|release|stats",
         "show heap usage info (available only if compiled with tcmalloc)"),
     MDSCommand("cache drop name=timeout,type=CephInt,range=0,req=false", "trim cache and optionally request client to release all caps and flush the journal"),
+    MDSCommand("scrub_path name=path,type=CephString name=scrubops,type=CephChoices,strings=force|recursive|repair,n=N,req=false",
```

Comment:
Let's be consistent by using spaces, so `scrub path`. Or, even better, we could adopt `scrub start` so it's harmonious with the new scrub commands you're adding below.

Reply:
I kept the command as-it-is -- I'm ok with `scrub start` since its much clear.

My only worry is that the asok interface is `scrub_path` and tell interface would be `scrub start` -- shouldn't the commands be consistent? (the asok command can be flipped to `scrub start` too)...

Reply:
I think the asok commands should just become deprecated.